comfy_cli/command/run/__init__.py

```
@@ -720,6 +738,23 @@ def execute_cloud(
     # its foreach item map to stash on the job state at submit time.
     compose_meta = pop_compose_meta(parsed_workflow)
 
+    # Cloud path uses cached/bundled object_info (no live server needed). Load
+    # it up front so checkpoint resolution can run BEFORE the preview/print
+    # below — the audit trail must advertise the graph we actually submit.
+    try:
+        from comfy_cli.cql.engine import _load_from_target
+
+        cloud_object_info = _load_from_target(mode="cloud")
+    except Exception:  # noqa: BLE001
```

Pylint (4.0.6)

[warning] Catching too general exception Exception

(W0718)